front/src/containers/SearchPage/SearchView2.tsx

```diff
@@ -465,6 +465,17 @@ class SearchView2 extends React.Component<SearchView2Props, SearchView2State> {
         return null;
     }
   };
+  handleLoadMore=()=>{
+    const { page, pageSize, sorts } = this.props.params;
+    // console.log("470", page, pageSize)
+    // pipe(changePage, this.props.onUpdateParams)(page + 1);
+    pipe(changePage, this.paginationHelper)(page + 1);
```

Comment:
This is how we handled loading the next page of results so I was trying to recreate that it didn't prove promising